packages/foreman/foreman-bootloaders-redhat/foreman-bootloaders-redhat.spec

```diff
@@ -44,29 +44,35 @@ install -Dp -m0755 %{SOURCE0} %{buildroot}%{_bindir}/foreman-generate-bootloader
 %{_bindir}/foreman-generate-bootloaders
 
 %files tftpboot
-%ghost %{_sharedstatedir}/tftpboot/grub2
-%ghost %{_sharedstatedir}/tftpboot/grub2/grubaa64.efi
-%ghost %{_sharedstatedir}/tftpboot/grub2/grubia32.0
-%ghost %{_sharedstatedir}/tftpboot/grub2/grubia32.efi
-%ghost %{_sharedstatedir}/tftpboot/grub2/grubppc64.efi
-%ghost %{_sharedstatedir}/tftpboot/grub2/grubppc64.elf
-%ghost %{_sharedstatedir}/tftpboot/grub2/grubppc64le.efi
-%ghost %{_sharedstatedir}/tftpboot/grub2/grubppc64le.elf
-%ghost %{_sharedstatedir}/tftpboot/grub2/grubppc.efi
-%ghost %{_sharedstatedir}/tftpboot/grub2/grubppc.elf
-%ghost %{_sharedstatedir}/tftpboot/grub2/grubx64.efi
-%ghost %{_sharedstatedir}/tftpboot/grub2/shim.efi
-%ghost %{_sharedstatedir}/tftpboot/grub2/shimx64.efi
-%ghost %{_sharedstatedir}/tftpboot/grub2/shimx64-redhat.efi
-%ghost %{_sharedstatedir}/tftpboot/grub2/shimaa64.efi
-%ghost %{_sharedstatedir}/tftpboot/grub2/shimaa64-redhat.efi
+%ghost %attr(644, root, root) %{_sharedstatedir}/tftpboot/grub2
+%ghost %attr(644, root, root) %{_sharedstatedir}/tftpboot/grub2/grubaa64.efi
+%ghost %attr(644, root, root) %{_sharedstatedir}/tftpboot/grub2/grubia32.0
+%ghost %attr(644, root, root) %{_sharedstatedir}/tftpboot/grub2/grubia32.efi
+%ghost %attr(644, root, root) %{_sharedstatedir}/tftpboot/grub2/grubppc64.efi
+%ghost %attr(644, root, root) %{_sharedstatedir}/tftpboot/grub2/grubppc64.elf
+%ghost %attr(644, root, root) %{_sharedstatedir}/tftpboot/grub2/grubppc64le.efi
+%ghost %attr(644, root, root) %{_sharedstatedir}/tftpboot/grub2/grubppc64le.elf
+%ghost %attr(644, root, root) %{_sharedstatedir}/tftpboot/grub2/grubppc.efi
+%ghost %attr(644, root, root) %{_sharedstatedir}/tftpboot/grub2/grubppc.elf
+%ghost %attr(644, root, root) %{_sharedstatedir}/tftpboot/grub2/grubx64.efi
+%ghost %attr(644, root, root) %{_sharedstatedir}/tftpboot/grub2/shim.efi
+%ghost %attr(644, root, root) %{_sharedstatedir}/tftpboot/grub2/shimx64.efi
+%ghost %attr(644, root, root) %{_sharedstatedir}/tftpboot/grub2/shimx64-redhat.efi
+%ghost %attr(644, root, root) %{_sharedstatedir}/tftpboot/grub2/shimaa64.efi
+%ghost %attr(644, root, root) %{_sharedstatedir}/tftpboot/grub2/shimaa64-redhat.efi
 
 
 %post tftpboot
 %{_bindir}/foreman-generate-bootloaders x86_64 noinstall
 
 
 %changelog
+* Wed Apr 24 2019 Lukas Zapletal <lzap+rpm@redhat.com> 201901011200-1
+- Fix BZ#1702434
+
+* Wed Mar 06 2019 Eric D. Helms <ericdhelms@gmail.com> - 201801241201-4
+- Fix BZ#1672498
```

Comment:
Yeah probably some mis-edit, should be `-3` there was not `-4`.

Reply:
Oh I am wrong again:

```
* Wed Mar 06 2019 Eric D. Helms <ericdhelms@gmail.com> - 201801241201-4
- Fix BZ#1672498

* Wed Apr 04 2018 Patrick Creech <pcreech@redhat.com> - 201801241201-3
- Satellite 6.4.0 rebuild

* Fri Feb 02 2018 Eric D. Helms <ericdhelms@gmail.com> 201801241201-2
- Fixes #22499 - fixed bootloader generator (lzap+git@redhat.com)

* Thu Jan 25 2018 Eric D. Helms <ericdhelms@gmail.com> 201801241201-1
- Foreman-bootloaders no longer ships binaries (lzap@teepee.home.lan)
```

Reply:
No, -3 was a downstream-only rebuild with no changes. As written in the original commit "revision is 4 instead of 3 to match downstream builds" :)

Reply:
It's kindof weird because we accidentaly maintained this file downstream only for few months. But it is just a changelog.